sys/riscv/riscv/machdep.c

```diff
@@ -1275,6 +1275,7 @@ bzero(void *buf, size_t len)
 	while(len-- > 0)
 		*p++ = 0;
 }
+
```

Comment:
Best to drop these blank lines before the CHERI CHANGES annotations when merging to dev.